text/0000-lts-releases.md

```diff
@@ -0,0 +1,109 @@
+# Meta
+[meta]: #meta
+- Name: Lifecycle releases with long term support (LTS)
+- Start Date: 2023-02-24
+- Author(s): joe-kimmel-vmw
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Lifecycle to define approximately 4 releases per year as LTS.
+LTS releases will have critical CVEs backported for a window of at least 1 year.
+LTS releases may also recieve other backports, including but not limited to golang patch version bumps.
+
+# Definitions
+[definitions]: #definitions
+
+**CVE**: While some scholars claim this acronym has origins in the Mayan codices[1], others seem to support a Norse origin[2]. Regardless of its hagiography, CVEs come in varying severities such as Critical, High, and "you probably don't need to patch it but we still wanted to let you know."
+These severity levels are issued by the shadowy council of security elders and cannot be modified by mere practitioners such as ourselves.
+However there are multiple councils[3] and they will occasionally grade the same vulnerability differently.
+This proposal only relates to CVEs graded Critical by at least one council.
+
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this? Some organizations that bundle lifecycle into other products are sufficiently risk-averse to be opposed to upgrading to the latest version, but still desire critical CVE patches. This LTS patch-backporting strategy allows us to support these slower moving vendors with minimal effort.
+- What use cases does it support? See above.
+- What is the expected outcome? Lifecycle will have to maintain a trailing list of supported LTS versions with approximate sunset dates for each. Then for certain CVEs there will be some additional "release engineering" effort to ease backporting and then releasing patch versions.
+
+# What it is
+[what-it-is]: #what-it-is
+
+It is risk mitigation for consumer and enterprise customers who want to apply critical patches without re-testing their entire platform for regressions which could occur during a full upgrade cycle.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+Lifecycle will maintain a publicly viewable table of LTS versions with anticipated sunset dates for support. 
+Not all Lifecycle releases will need to be LTS, but 3-5 each year probably will be marked as LTS. Maintainers will be responsible for marking releases as LTS.
+Some additional release automation will be built to support patching and releasing LTS releases.
```

Comment:
Potentially:
* Automation around release notes
* Some system where tagging a PR as applicable to an LTS release creates new PRs against the LTS tags

It's probably not necessary to put this in the RFC, I'm just thinking out loud here